cli/cmd/utils.go

```diff
@@ -141,7 +142,31 @@ func doesKubeconfigExist(*cobra.Command, []string) error {
 }
 
 func getLokoConfig() (*config.HCLConfig, hcl.Diagnostics) {
-	return config.LoadConfig(viper.GetString("lokocfg"), viper.GetString("lokocfg-vars"))
+	lokocfgFiles, diags := config.LoadHCLFiles(viper.GetString("lokocfg"), "lokocfg")
```

Comment:
@invidian I agree, this would be second part of the refactoring. Where everything related to loading and parsing of Lokomotive configuration and management of Lokomotive cluster would be moved from `cli/cmd` package to `pkg/config` and `pkg/cluster` packages.

I have purposefully not touched upon any changes regarding the above so as to keep the changes to a minimal and the scope of the PR smaller.